src/server/pachyderm_test.go

```diff
@@ -10620,11 +10621,11 @@ func TestDatabaseStats(t *testing.T) {
 		hdr, err := f.Header()
 		require.NoError(t, err, "getting database tar file header should succeed")
 		require.NotMatch(t, "^[a-zA-Z0-9_\\-\\/]+\\.error$", hdr.Name)
-
 		if hdr.Name == "database/row-counts.json" {
```

Comment:
This wants to be a switch hdr.Name { case ... } statement now.

Reply:
True!